salt/modules/pacman.py

```diff
@@ -686,7 +684,7 @@ def upgrade(refresh=False, root=None, **kwargs):
             and __salt__['config.get']('systemd.scope', True):
         cmd.extend(['systemd-run', '--scope'])
     cmd.extend(['pacman', '-Su', '--noprogressbar', '--noconfirm'])
-    if salt.utils.is_true(refresh):
+    if refresh:
```

Comment:
If we're not using `salt.utils.is_true()` any more, then we should be using `if refresh is True` here.